core_lib/data_layers/data/handler/sql_alchemy_data_handler_registry.py

```diff
@@ -29,13 +27,12 @@ def connection(self):
 
     def get(self, use_parent_instance=False, *args, **kwargs) -> SqlAlchemyDataHandler:

```

Comment:
1. read the code 
2. try to understand the code 
3. delete the `use_parent_instance` + the first portion of the if statment